docs/additional-functionality/ml-integration.md

```diff
@@ -36,3 +36,9 @@ val maxValue = rdd.map(table => {
 You may need to disable RMM caching when exporting data to an ML library as that library
 will likely want to use all of the GPU's memory and if it is not aware of RMM it will not have
 access to any of the memory that RMM is holding.
+
+## Spark ML Algorithms Supported by RAPIDS Accelerator
+
+The plugin implements the `transform` API for [Principal Component Analysis (PCA)](https://spark.apache.org/docs/latest/mllib-dimensionality-reduction#principal-component-analysis-pca) as an [accelerated UDF](https://github.com/NVIDIA/spark-rapids/blob/branch-21.12/docs/additional-functionality/rapids-udfs.md#rapids-accelerated-user-defined-functions).
+
+You can find the details in [spark-rapids-ml](https://github.com/NVIDIA/spark-rapids-ml) and the use case in [spark-rapids-example's pca subfolder](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-21.10/examples/pca).
```

Comment:
Lines should be wrapped at approx 100 characters to make viewing in a terminal and identifying changes per line easier.

Also the `transform` API is _not_ implemented by the RAPIDS Accelerator, it's done as a separate example.  All the RAPIDS Accelerator does here is provide the RAPIDS accelerated UDF framework.  All the stuff specific to PCA is not provided directly.

Suggest something more like:
```suggestion
The [spark-rapids-examples repository](https://github.com/NVIDIA/spark-rapids-examples) provides a
[working example](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-21.10/examples/pca)
of accelerating the `transform` API for
[Principal Component Analysis (PCA)](https://spark.apache.org/docs/latest/mllib-dimensionality-reduction#principal-component-analysis-pca).
The example leverages the [RAPIDS accelerated UDF interface](rapids-udfs.md) to provide a native
implementation of the algorithm. The details of the UDF implementation can be found in the
[spark-rapids-ml repository](https://github.com/NVIDIA/spark-rapids-ml).
```